modin/pandas/io.py

```diff
@@ -761,7 +761,7 @@ def read_orc(
     Load an ORC object from the file path, returning a DataFrame.
     """
     ErrorMessage.default_to_pandas("read_orc")
-    return DataFrame(pandas.read_orc(path, columns, **kwargs))
+    return DataFrame(pandas.read_orc(path, columns=columns, **kwargs))
```

Comment:
This is not a mistake, it's just more convenient to test, from the point of view of logic, nothing changes.